documentation/using/master.adoc

```diff
@@ -25,16 +23,14 @@ include::assemblies/tracing/assembly-distributed-tracing.adoc[leveloffset=+1]
 
 include::assemblies/security/assembly-security.adoc[leveloffset=+1]
 
-//managing part
-include::assemblies/managing/assembly-management-tasks.adoc[]
+include::assemblies/managing/assembly-management-tasks.adoc[leveloffset=+1]
 
 [id='api_reference-{context}']
 :parent-context: {context}
 :context: reference
 //schema part
-= Custom resource API reference
+== Custom resource API reference
 
-include::modules/con-common-configuration-properties.adoc[]
-== Schema properties
```

Comment:
I did like this heading. Does it need to go?

Reply:
I didn't think we needed it, but now it's back I think I prefer to have it! One issue, which might not be much of one, is that we now have 4 heading levels when we go down to property descriptions.

![image](https://user-images.githubusercontent.com/47596553/111348358-06032900-8678-11eb-87ed-188b041f1d19.png)

Reply:
Ah, I didn't consider that. We should avoid any extra nesting of headings. Sorry about that!

Reply:
But I don't mind if you want to merge into `master` and see how it looks.

Reply:
Let's merge it in